test/parallel/test-internal-util-helpers.js

```diff
@@ -0,0 +1,32 @@
+// Flags: --expose-internals
+'use strict';
+
+require('../common');
+const assert = require('assert');
+const { types } = require('util');
+const { isError } = require('internal/util');
```

Comment:
I am not sure if testing the internals is the best way to do this, since this simply serves other APIs...can't we test util.inspect or the repl instead?

Reply:
I could write specific tests for those but in that case we would have to add specific tests for each using API. I would rather just add a comment why we test the functionality here. Is that ok?